en/websites_privacy_notice.md

```diff
@@ -1,26 +1,27 @@
 
-* **Contributors**: Volunteering for Mozilla as a community contributor may require Mozilla and others to communicate with you at the email address that you provide in connection to your contribution and to recognize your efforts. If you contribute to Bugzilla, Mozilla Reps, or our code base, then your email address and possibly your name will be publicly available to all internet users. If you create an account on Mozilla Connect (platform powered by [Khoros](https://khoros.com/privacy), which has its own privacy notice), your name, your avatar, your posts and replies and other information you share will be accessible to Mozilla and other community members. If you create a profile at people.mozilla.org, it will be accessible to Mozilla employees and Mozilla contributors. You can edit your profile data at [Profile Settings](https://people.mozilla.org/e?section=personal-info). We sometimes use contributor information from sources (such as Bugzilla) in dashboards to visually share aggregated data on the Mozilla community. An example is [https://wiki.mozilla.org/Contribute/Dashboards](https://wiki.mozilla.org/Contribute/Dashboards). If possible, we try to minimize contact information that is publicly displayed.
+* **Contributors**: Volunteering for Mozilla as a community contributor may require Mozilla and others to communicate with you at the email address that you provide in connection to your contribution and to recognize your efforts. If you contribute to Bugzilla, or our code bases, then your email address and possibly your name will be publicly available to all internet users. If you create an account on Mozilla Connect (platform powered by [Khoros](https://khoros.com/privacy), which has its own privacy notice), your name, your avatar, your posts and replies and other information you share will be accessible to Mozilla and other community members. If you create a profile at people.mozilla.org, it will be accessible to Mozilla employees and Mozilla contributors; you can edit your profile data at [Profile Settings](https://people.mozilla.org/e?section=personal-info). We sometimes use contributor information from sources (such as Bugzilla) in dashboards to visually share aggregated data on the Mozilla community. An example is [https://wiki.mozilla.org/Contribute/Dashboards](https://wiki.mozilla.org/Contribute/Dashboards). Where possible, we try to minimize contact information that is publicly displayed.
 
 * **User Feedback**: You can provide feedback to us on our products and services on webpages like [input.mozilla.org](https://input.mozilla.org/), through an in-product experience, or through channels such as email, Bugzilla, Matrix, a social media account, our Get Involved page, or through a group like Student Ambassadors. Please minimize the personal information you choose to share on these forums because your comments may be accessible to the public.
```

Comment:
```suggestion
* **User Feedback**: You can provide feedback to us on our products and services on webpages like [connect.mozilla.org](https://connect.mozilla.org/), through an in-product experience, or through channels such as email, Bugzilla, Matrix, a social media account, our Get Involved page. Please minimize the personal information you choose to share on these forums because your comments may be accessible to the public.
```

https://input.mozilla.org/ is long gone and replaced by connect.mozilla.org
Student Ambassadors was shut down a long time ago as well.